frontend/yarn.lock

```diff
@@ -1,10452 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-# yarn lockfile v1
```

Comment:
Should not be deleted

Reply:
My bad, i deleted on my local to refresh dependencies (for eslint) and i forgot to commit it.